dspace-server-webapp/src/main/java/org/dspace/app/rest/RestResourceController.java

```diff
@@ -754,7 +755,7 @@ public ResourceSupport findRel(HttpServletRequest request, HttpServletResponse r
                                                                            String id, String rel, String relid,
                                                                            Pageable page,
                                                                            PagedResourcesAssembler assembler,
-                                                                           String projection) {
+                                                                           String projection) throws Throwable {
```

Comment:
I guess `throws Throwable` was left here during early developments

Reply:
This is explained at https://github.com/atmire/DSpace/blob/authority-control-bugfixes/dspace-server-webapp/src/main/java/org/dspace/app/rest/RestResourceController.java#L779